databricks_sync/sdk/terraform.py

```diff
@@ -88,7 +88,7 @@ def _cmd(self, cmds, *args, **kwargs):
         # Close buffers
         try:
             p.stdout.flush()
-        except:
+        except Exception:
             pass
         p.stdout.close()
```

Comment:
```suggestion
        except Exception as e:
            print(str(e))
        finally:
            p.stdout.close()
```

Reply:
@R7L208 can you switch the print statement to log.debug?

Reply:
Updated @stikkireddy